starter-code/iteration-1/main.ts

```diff
@@ -1,12 +1,21 @@
 // Iteration 1
 // 1. Create an array of strings
-
+var tasks: Array<string> = [];
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
-
+function addTask(task: string):void {
+    tasks.push(task);
+};
 // 3. Create a function to list all tasks, it must show in the console de task.
+function listAllTasks():void {
+    tasks.forEach((task: string)=>{
+        console.log(task);
+    });
+};
 
 // 4. Create a function to delete a task, you must find the task inside the array and delete it.
-
+function deleteTask(chosenTask) {
```

Comment:
this function should expect a number
`function deleteTask (task: string): number {`